Formula/freealut.rb

```diff
@@ -37,7 +38,7 @@ def install
 --- a/configure.ac
 +++ b/configure.ac
 @@ -83,7 +83,8 @@ AC_DEFINE([ALUT_BUILD_LIBRARY], [1], [Define to 1 if you want to build the ALUT
- 
+
```

Comment:
Does this break the patch?

Reply:
interesting, that must get slipped in with my vscode auto-format

Reply:
It wont break the change, but I will have this excluded for the PR. Nice catch.

Reply:
Just wondering because I saw a few of these in a similar PR

Reply:
That might also relies on we do the patch in the `patch` block. Let me read the code too.

Reply:
DataPatch is in here, https://github.com/Homebrew/brew/blob/master/Library/Homebrew/patch.rb#L80-L101

let me quickly test the whitespace strip effect in my local.

Reply:
yeah, as expected, patching works fine (with the whitespace stripped out).
```
==> Patching
patching file configure.ac
==> ./autogen.sh
configure.ac:78: warning: The macro `AC_LIBTOOL_WIN32_DLL' is obsolete.
```

Reply:
I just tested it locally (although not extensively). It seems to work fine.

Reply:
Oops, I hadn't loaded the most recent comments... I concur 👍